build.gradle

```diff
@@ -247,13 +250,30 @@ project(':iceberg-data') {
       exclude group: 'org.slf4j', module: 'slf4j-log4j12'
     }
 
-    testCompile("org.apache.hadoop:hadoop-client") {
+    implementation("org.apache.orc:orc-core::nohive") {
+      exclude group: 'org.apache.hadoop'
+      exclude group: 'commons-lang'
+      // These artifacts are shaded and included in the orc-core fat jar
+      exclude group: 'com.google.protobuf', module: 'protobuf-java'
```

Comment:
Is it intended for the protobuf-java class coming from ORC, to be part of the runtime jar ?
For example: org/apache/iceberg/shaded/org/apache/orc/protobuf/CodedInputStream$ArrayDecoder.class